dnf5/commands/download/download.cpp

```diff
@@ -86,11 +88,30 @@ void DownloadCommand::set_argument_parser() {
     url->set_const_value("true");
     url->link_value(url_option);
 
+    urlprotocol_valid_options = {"http", "https", "rsync", "ftp"};
```

Comment:
"rsync" should be ignored  according to #497.

Reply:
Remove "rsync" also from help text in `urlprotocol->set_arg_value_help("{http, https, rsync, ftp},...")`.

Reply:
Would "file" also be a valid option?

Reply:
Yes, keep "file" among remote protocols. We currently handle file:// schema as a remote location (see issue #624).